nifi-nar-bundles/nifi-marklogic-bundle/nifi-marklogic-services/src/test/java/org/apache/nifi/marklogic/controller/DefaultMarkLogicDatabaseClientServiceIT.java

```diff
@@ -44,8 +47,16 @@ public void testService() throws InitializationException {
         runner.setProperty(service, DefaultMarkLogicDatabaseClientService.USERNAME, username);
         runner.setProperty(service, DefaultMarkLogicDatabaseClientService.PASSWORD, password);
         runner.setProperty(service, DefaultMarkLogicDatabaseClientService.SECURITY_CONTEXT_TYPE, authentication);
+        runner.setProperty(service, DefaultMarkLogicDatabaseClientService.LOAD_BALANCER, loadBalancer);
         runner.enableControllerService(service);
         runner.assertValid(service);
+        runner.disableControllerService(service);
+        try {
+            service.getDatabaseClient().newBinaryDocumentManager().read("/test.json").getPageSize();
+            Assert.assertTrue("Should not be able to use client after shutdown", false);
```

Comment:
I think Assert.fail would be more clear here. 

```suggestion
            Assert.fail("Should not be able to use client after shutdown");
```